hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -35,12 +43,24 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
 If you wish, you can remove the `program_usb_boot_mode` line from config.txt, so that if you put the SD card into another Raspberry Pi, it won't program USB host boot mode. Make sure there is no blank line at the end of config.txt.
```

Comment:
@JamesH65 Is the bit about "Make sure there is no blank line at the end of config.txt." true / relevant?